### PR TITLE
Update SETUP.md for Mac OS X Catalina

### PR DESCRIPTION
**Pull request by davidsbailey**

### CODE REVIEW

File: SETUP.md [hunk 1]
```diff
@@ -82,8 +93,12 @@ After setup, read about our [code styleguide](./STYLEGUIDE.md), our [test suites
           GIT_COMPLETION_CHECKOUT_NO_GUESS=1
           autoload -Uz compinit && compinit
           ```
+        * fix any problems with compinit:
+          ```
+          compaudit | xargs chmod g-w
+          ```
     </details>
-1. Install Homebrew: `ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"`
+1. Install Homebrew: `/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"`
```
Comment (davidsbailey) [inline SETUP.md:101]: when I ran the old command, it told me it was deprecated and that I should run this command instead.

File: SETUP.md [hunk 2]
```diff
@@ -6,16 +6,16 @@ You can do Code.org development using OSX, Ubuntu, or Windows (running Ubuntu in
...
 
      ```
      ruby --version  # --> ruby 2.5.0
      node --version  # --> v8.15.0
      yarn --version  # --> 1.16.0
      ```
-1. If using HTTPS: `git clone https://github.com/code-dot-org/code-dot-org.git`, if using SSH: `git@github.com:code-dot-org/code-dot-org.git`
-1. `gem install bundler -v 1.17`
+1. If using SSH (recommended): `git clone git@github.com:code-dot-org/code-dot-org.git` , if using HTTPS: `git clone https://github.com/code-dot-org/code-dot-org.git`, 
+1. `gem install bundler -v 1.17.3`
```
Comment (davidsbailey) [inline SETUP.md:18]: updating this to match the version in our Gemfile: https://github.com/code-dot-org/code-dot-org/blob/a4fbf71b87dab7363a0a35314002eea39e13f22d/Gemfile.lock#L1075-L1076

File: SETUP.md [hunk 3]
```diff
@@ -38,6 +38,12 @@ You can do Code.org development using OSX, Ubuntu, or Windows (running Ubuntu in
 
 1. `bundle exec rake install`
     * This can take a long time, ~30 minutes or more. The most expensive are the "seeding" tasks, where your local DB is populated from data in the repository. Some of the seeding rake tasks can take several minutes. The longest one, `seed:scripts`, can take > 10 minutes, but it should at least print out progress as it goes.
+
+1. fix your database charset and collation to match our servers
+    * `bin/dashboard-sql`
+    * `ALTER DATABASE dashboard_development CHARACTER SET utf8 COLLATE utf8_unicode_ci;`
+    * `ALTER DATABASE dashboard_test CHARACTER SET utf8 COLLATE utf8_unicode_ci;`
```
Comment (davidsbailey) [inline SETUP.md:45]: The long term solution here is a bit uncertain, but the consequences of not doing this during setup right now are that you risk ending up with diffs in schema.rb which show up after each time you do `rake build` or `rake db:migrate`. Much inconclusive discussion [here](https://codedotorg.slack.com/archives/C0T10HG6N/p1601513126001300) and [here](https://codedotorg.slack.com/archives/C0T0PNTM3/p1603225836458900).

Comment (davidsbailey) [reply SETUP.md:45]: @jamescodeorg @sureshc @wjordan this is the workaround I'm proposing in the short term to deal with character set and collation issues on developer machines.

**Review by jamescodeorg — APPROVED** [review]
Looks great, thanks for making these updates!

**Review by dmcavoy — APPROVED** [review]

**Pull request merged**